nodemongo/controller/employees.js

```diff
@@ -0,0 +1,67 @@
+
+
+var express = require('express');
+var bodyParser = require('body-parser');
+var multer = require('multer'); 
+var upload = multer(); 
+var app = require('express')();
+app.use(bodyParser.json()); 
+app.use(bodyParser.urlencoded({ extended: true })); 
+var router = express.Router();
+var request = require('request');
+var http = require('http');
+var employeeModel = require('../model/employeeModel.js');
+module.exports = router;
+
+
+
+router.get('/', function(res, req, next){
```

Comment:
@psinha20 Indentation and the variable naming convention is not proper. 

**Why you're repeating same mistakes again and how many times I need to write same comments?**

cc @ketanTechracers

Reply:
@psinha20 Use space after 'function()'

**Same mistake.**

Reply:
@psinha20 Where is your package.json file? How can I run your code?

**Remember** two things when creating any node project -
- npm init (It will create the package.json file)
- Installing dependencies with '--save' flag i.e npm install --save 'name'

cc @ketanTechracers Please look into this. He is doing the same mistakes again like indentation, variable naming convention, package.json changes and unnecessary package import.